### PR TITLE
Scp3144 severed hands fix

### PR DESCRIPTION
**Pull request by Undid-Iridium**
Scp3144 severed hands fix - probably

https://github.com/orgs/ExMod-Team/projects/1/views/1?pane=issue&itemId=88769498&issue=ExMod-Team%7CEXILED%7C197
<br />

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [] I have checked no IL patching errors in the console 

### Other
- [X] Still requires more testing

https://github.com/orgs/ExMod-Team/projects/1/views/1?pane=issue&itemId=88769498&issue=ExMod-Team%7CEXILED%7C197

### CODE REVIEW

File: EXILED/Exiled.API/Features/DamageHandlers/GenericDamageHandler.cs [hunk 1]
```diff
@@ -90,6 +90,9 @@ public GenericDamageHandler(Player player, Player attacker, float damage, Damage
                 case DamageType.SeveredHands:
                     Base = new UniversalDamageHandler(damage, DeathTranslations.SeveredHands, cassieAnnouncement);
                     break;
+                case DamageType.SeveredEyes:
+                    Base = new UniversalDamageHandler(damage, DeathTranslations.SeveredHands, cassieAnnouncement);
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/DamageHandlers/GenericDamageHandler.cs:94]: Wrong DeathTranslation

File: EXILED/Exiled.API/Enums/DamageType.cs [hunk 2]
```diff
@@ -120,6 +120,11 @@ public enum DamageType
         /// </summary>
         SeveredHands,
 
+        /// <summary>
+        /// Damage caused by severed eyes.
+        /// </summary>
+        SeveredEyes,
+
```
Comment (louis1706) [inline EXILED/Exiled.API/Enums/DamageType.cs:127]: Replace the Scp1344 with this one

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.API/Enums/DamageType.cs (hunk 2 above)
Comment (Undid-Iridium) [reply EXILED/Exiled.API/Enums/DamageType.cs:127]: Double check me

Comment (louis1706) [reply EXILED/Exiled.API/Enums/DamageType.cs:127]: Sorry i meaned than Scp1344 damage is not more used so replace the name of damagetype with the new one you made

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Enums/DamageType.cs:127]: So good?

Comment (louis1706) [reply EXILED/Exiled.API/Enums/DamageType.cs:127]: Just removed line 261 no ?

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Enums/DamageType.cs:127]: Done

**Pull request merged**